crawl-ref/source/tilepick.cc

```diff
@@ -2849,7 +2849,7 @@ tileidx_t tileidx_item(const item_def &item)
 #endif
 
     case OBJ_CORPSES:
-        if (item.sub_type == CORPSE_SKELETON)
+        if (!Options.show_blood || item.sub_type == CORPSE_SKELETON)
```

Comment:
Maybe could be merged without this, but I do agree with dilly's comment that this distinction matters for gameplay purposes if you're playing a character using certain spells/abilities, so it would be best to have some other indicator of skeleton vs corpse if at all possible. The most obvious thing might be to use a `TILE_FLAG..`, but I don't see any that could be repurposed.

Other brainstorming: I could also imagine recoloring the skeleton tiles, though I guess you are trying to not depict even the color of blood. Or some kind of simple visual tweak to the skele tiles? For reference, console distinguishes them pretty simply, like this: `†÷`, so whatever's here wouldn't need to be super perfect, and doesn't need to implement all the body shapes for the corpse-skele case IMO.

Also, extremely minor, but there's some flavor stuff that `_tileidx_corpse` does that is unrelated to blood.

Reply:
Thanks for the feedback, @rawlins. I will experiment with some alternatives on how to represent the corpses.